hazelcast/src/test/java/com/hazelcast/map/impl/nearcache/TestReadOnlyProcessor.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.map.impl.nearcache;
+
+import com.hazelcast.core.ReadOnly;
+import com.hazelcast.map.EntryProcessor;
+import org.jetbrains.annotations.Nullable;
```

Comment:
not sure if it is ok to do that in tests, but we were trying to use the annotations from `javax.annotation`. I belive it would be good to use it here as well

Reply:
Done